.travis.yml

```diff
@@ -26,6 +32,7 @@ before_install:
   - travis_retry composer self-update && composer --version
 
 install:
+  - if [ "$SYMFONY_VERSION" != "" ]; then travis_retry composer require "symfony/symfony:${SYMFONY_VERSION}" --prefer-source --no-update -n; fi;
```

Comment:
No, the `require` writes the change to the `composer.json` file.

See also: https://docs.travis-ci.com/user/languages/php/#Testing-Against-Multiple-Versions-of-Dependencies-(e.g.-Symfony)